### ISSUE TITLE
Release 2020.1.3 doesn't support Go 1.14's overlapping interfaces

### ISSUE THREAD

**Issue by mvdan** [dominikh/go-tools#706]
```
$ cat f.go
package p

type I1 interface {
	Foo()
	I1()
}

type I2 interface {
	Foo()
	I2()
}

type Overlapping interface {
	I1
	I2
}
$ go version
go version devel +d774d979dd Sat Mar 14 07:03:15 2020 +0000 linux/amd64
$ ./staticcheck/staticcheck --version
staticcheck 2020.1.3
$ ./staticcheck/staticcheck f.go
f.go:15:2: Foo redeclared (compile)
$ staticcheck --version
staticcheck (devel, v0.0.1-2020.1.0.20200222115104-c702d0a1c8d5)
$ staticcheck f.go
```

**Comment by mvdan** [comment]
I suspect the problem is that the release was built with Go 1.13.x, not 1.14.x, where `go/types` has support for this new feature.

**Comment by mvdan** [comment]
Also, to clarify, this is the 2020.1.3 binary release. If I was building it from source with Go 1.14.x or master, I assume this wouldn't fail.

**Comment by mvdan** [comment]
One last thought - given how important the Go version used to build staticcheck is, I think `--version` should include it, i.e. https://golang.org/pkg/runtime/#Version.

**Comment by dominikh** [comment]
> I suspect the problem is that the release was built with Go 1.13.x, not 1.14.x, where go/types has support for this new feature.

Correct. We'll need to make a new release.

> One last thought - given how important the Go version used to build staticcheck is, I think --version should include it, i.e. https://golang.org/pkg/runtime/#Version.

That information is available via `--debug.version`, which shows detailed information on how the binary was built.

**Comment by dominikh** [comment]
This should be fixed in 2020.1.4.
